test/suites/alpha/integration/extended_resources_test.go

```diff
@@ -488,3 +606,181 @@ func ExpectHabanaDevicePluginCreated() {
 		},
 	})
 }
+
+func ExpectNeuronDevicePluginCreated() {
+	GinkgoHelper()
+	env.ExpectCreated(&rbacv1.ClusterRole{
```

Comment:
nit/style:Do these all need to be separate `ExpectCreated` calls or should you define all the resources and then make a call to create them all?

Reply:
I don't see any issue with that. I'm happy to make that change, but is there any reason that's preferred stylistically?